pkg/agent/proxy/proxier.go

```diff
@@ -48,8 +50,10 @@ const (
 	resyncPeriod  = time.Minute
 	componentName = "antrea-agent-proxy"
 	// SessionAffinity timeout is implemented using a hard_timeout in OVS. hard_timeout is
-	// represented by a uint16 in the OpenFlow protocol,
+	// represented by a uint16 in the OpenFlow protocol.
 	maxSupportedAffinityTimeout = math.MaxUint16
+	// The prefix added to the multi-cluster Service name.
```

Comment:
```suggestion
	// The prefix of multi-cluster Service name.
```

Reply:
multi-cluster? Multi-cluster? Which should we use?

Reply:
when we talk about Antrea Multi-cluster feature, we will use `Multi-cluster`, if it's just referring to multi-cluster related concept, we will use `multi-cluster`.